docs/src/developers_guide/contributing_pytest_conversions.rst

```diff
@@ -50,6 +50,10 @@ Conversion Checklist
 #. Check for references to ``self.tmp_dir``. In pytest, ``tmp_path`` is used instead,
    and can be passed into functions as a fixture.
 #. Check for ``if __name__ == 'main'``. This is no longer needed with pytest.
+#. Remove the top-level import of :mod:`iris.tests` (usually ``import iris.tests as tests``).
+   Having followed the above steps, any remaining calls 
+   (e.g. :func:`iris.tests.get_data_path`) should be easily replacable with calls to
+   :mod:`iris.tests._shared_utils` (e.g. :func:`iris.tests._shared_utils.get_data_path`).
```

Comment:
This feels like it would make sense to do earlier on? Maybe step 2? It doesn't make much of a difference, but feels more linear.

Reply:
I did have it at step 2, but then it pollutes the IDE with hundreds of errors if you remove it so early, and depending on how you work this can be quite unhelpful.